src/pybind/mgr/dashboard/module.py

```diff
@@ -242,6 +243,11 @@ class Module(MgrModule, CherryPyConfig):
             "desc": "Create self signed certificate",
             "perm": "w"
         },
+        {
```

Comment:
By using `@CLICommand` decorator helper you may get this registered, a mapping between the command signature and the function arguments, and the command handler itself.

Reply:
Hmm, I was just following the pattern I saw. I do see now that the feature toggles have started using `CLICommand`; do you think we should aim to use that for all the commands we register?

Reply:
Well, currently there's still legacy code not using that, but most new code is using it, even in other mgr modules like Orchestrators (basically because of its benefits). I think we should use everything that increases the locality of changes: before all these new helpers we needed to add new MODULE_OPTIONS here, new COMMANDS there, the new command handling stuff over there, the endpoint controller in another file... That was my intent with pluggable architecture, but that's totally open to debate ;-)